components/hackathon/common/ScheduleSection.tsx

```diff
@@ -4,46 +4,21 @@ import { BiRightArrowAlt } from "@react-icons/all-files/bi/BiRightArrowAlt";
 import { useTrack } from "hooks/analytics/useTrack";
 import { Badge, Heading, Text } from "tw-components";
 
-export const ScheduleSection: React.FC = () => {
+interface IScheduleSection {
+  scheduleItems: {
+    day: number;
+    title: string;
+    href: string;
+    irl?: string;
+  }[];
+  month: string;
+}
```

Comment:
same comment about the naming of the interface and extracting or array syntax for the items